.coreos/desoto.env.dist

```diff
@@ -1,7 +1,6 @@
 # Environment variables for the DeSoto service discovery
 # This corresponds to an etcd entry at /environments/desoto
 
-ETCD_HOSTS=http://172.17.42.1:4001
```

Comment:
I understand this value is set at launch time by the `desoto.service` file but wouldn't it be good to keep the key commented out in this file, maybe with a comment, so that people are not confused if they expected it to be here?